README.md

```diff
@@ -187,6 +187,21 @@ The cohort endpoints return JSON in this format:
 ## [Register an Application](#register)
 Census uses [Devise](https://github.com/plataformatec/devise), [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) and [cancancan](https://github.com/CanCanCommunity/cancancan) to manage authentication and authorization.
 
+To Get a token, register your app through Census and then make a request:
+
+Source [here.](https://github.com/doorkeeper-gem/doorkeeper/wiki/Client-Credentials-flow)
+
+```javascript
+conn.post do |req|
+  req.url '/oauth/token'
+  req.params['grant_type'] = 'client_credentials'
+  req.params['client_id'] = ENV['CENSUS_CLIENT_ID'] #=> To be supplied by the developer
+  req.params['client_secret'] = ENV['CENSUS_SECRET_ID'] #=> To be supplied by the developer
+end
```

Comment:
2. This is `ruby` not `javascript`
1. `supplied by the developer` is technically correct, but in the context of "Register an Application", I would say something like `provided by census interface`
3. You might mention that this is using `Faraday`, but that any HTTP library will work